src/UI/TransactionsToastList/components/CustomToast/CustomToast.tsx

```diff
@@ -11,6 +13,19 @@ export const CustomToast = (props: CustomToastPropsType) => {
     return <CustomComponentToast {...props} />;
   }
 
+  if (props.transaction) {
+    const transaction = props.transaction as any;
```

Comment:
I'd use `SignedTransactionType` instead of any

Reply:
I tried, but it throws errors. The `TransactionToast` and `CustomToast` accespt both `SignedTransactionType` and `ServerTransactionType` which have different implementation of `transaction`. It complicates things very much to change the types and not worth it. Using simply `any` here does the job without too much hustle

Reply:
`SignedTransactionType | ServerTransactionType`. If you want to cast to a specific type you can use `as unknown as ServerTransactionType`.
A specific type is desired because you know what to expect from that variable when you read the code first time. 'any' should be used just in exceptional situations because it breaks the code safeness